### PR TITLE
develop < i18n

### PR DESCRIPTION
**Pull request by JakeMakesStuff**

### CODE REVIEW

File: magiccap/i18n/en/app.po [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# app.js:40
+msgid "Application"
```
Comment (vladfrangu) [inline magiccap/i18n/en/app.po]: This is wrong on many levels. The ID should be something unique and easy to write ("app.title" for instance) while the string should be the actual translation

File: magiccap/i18n/index.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+// Gets the language pack information.
+const nativeLangNames = require("./native_lang_names.json");
+const { readFile } = require("fs-nextra");
+const { readdirSync, statSync } = require("fs");
+const { join } = require("path");
+const PO = require("pofile");
+
+// Used to cache/get *.po files.
+const poCache = {};
```
Comment (vladfrangu) [inline magiccap/i18n/index.js]: This should be a map

File: magiccap/i18n/index.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+			.replace(/"/g, "&quot;")
+			.replace("{license}", '<a href="javascript:openMPL()">MPL-2.0</a>')
+			.replace("{acceleratorDocs}", `<a href="javascript:openAcceleratorDocs()">${i18nThisDocumentation}</a>`);
+		htmlDone = htmlDone.replace(regexParse[0], poParsed);
+	}
+	return htmlDone;
+};
+
+// Handles showing all of the language packs.
+const langPackInfo = {};
```
Comment (vladfrangu) [inline magiccap/i18n/index.js]: Map

File: magiccap/i18n/en/app.po (hunk 1 above)
Comment (JakeMakesStuff) [reply magiccap/i18n/en/app.po]: It shouldn't. Services such as CrowdIn use po files ID's for the translation.

**Pull request merged**